packages/paste-core/primitives/box/src/index.tsx

```diff
@@ -77,37 +78,41 @@ export interface BoxStyleProps
   float?: FloatProperty;
   willChange?: WillChangeProperty;
   clip?: ClipProperty;
+  opacity?: string;
 }
 
-interface PseudoStylesProps {
-  _after?: BoxStyleProps;
-  _before?: BoxStyleProps;
-  _focus?: BoxStyleProps;
-  _hover?: BoxStyleProps;
-  _active?: BoxStyleProps;
-  _pressed?: BoxStyleProps;
-  _selected?: BoxStyleProps;
-  _focusWithin?: BoxStyleProps;
-  _invalid?: BoxStyleProps;
-  _disabled?: BoxStyleProps;
-  _grabbed?: BoxStyleProps;
-  _expanded?: BoxStyleProps;
-  _checked?: BoxStyleProps;
-  _mixed?: BoxStyleProps;
-  _odd?: BoxStyleProps;
-  _even?: BoxStyleProps;
-  _visited?: BoxStyleProps;
-  _readOnly?: BoxStyleProps;
-  _first?: BoxStyleProps;
-  _last?: BoxStyleProps;
-  _groupHover?: BoxStyleProps;
-  _notFirst?: BoxStyleProps;
-  _notLast?: BoxStyleProps;
-  _placeholder?: BoxStyleProps;
+export interface BoxPseudoStyleProps {
+  _after?: BoxBaseStyleProps;
+  _before?: BoxBaseStyleProps;
+  _focus?: BoxBaseStyleProps;
+  _hover?: BoxBaseStyleProps;
+  _active?: BoxBaseStyleProps;
+  _pressed?: BoxBaseStyleProps;
+  _selected?: BoxBaseStyleProps;
+  _focusWithin?: BoxBaseStyleProps;
+  _invalid?: BoxBaseStyleProps;
+  _disabled?: BoxBaseStyleProps;
+  _grabbed?: BoxBaseStyleProps;
+  _expanded?: BoxBaseStyleProps;
+  _checked?: BoxBaseStyleProps;
+  _mixed?: BoxBaseStyleProps;
+  _odd?: BoxBaseStyleProps;
+  _even?: BoxBaseStyleProps;
+  _visited?: BoxBaseStyleProps;
+  _readOnly?: BoxBaseStyleProps;
+  _first?: BoxBaseStyleProps;
+  _last?: BoxBaseStyleProps;
+  _groupHover?: BoxBaseStyleProps;
+  _notFirst?: BoxBaseStyleProps;
+  _notLast?: BoxBaseStyleProps;
+  _placeholder?: BoxBaseStyleProps;
+  __moz_focus_inner?: BoxBaseStyleProps;
```

Comment:
Added this as it was being used in CSS directly before.

Reply:
You'll need to add this to here https://github.com/twilio-labs/paste/blob/main/packages/paste-core/primitives/box/src/PseudoPropStyles.ts

Reply:
In fact I think if you remove `: {[key: string]: string}` from `PseudoPropStyles` in https://github.com/twilio-labs/paste/blob/main/packages/paste-core/primitives/box/src/PseudoPropStyles.ts You might be able to dynamically create this interface like 

```
type PseudoSelectors = keyof typeof PseudoPropStyles;
type PseudoStylesProps = {
  [key in PseudoSelectors]: BoxStyleProps;
}```

so we don't have to keep these in sync

Reply:
Good idea, fixed!